config/samples/neutron_v1beta1_neutronapi.yaml

```diff
@@ -17,4 +17,4 @@ spec:
   preserveJobs: false
   containerImage: quay.io/podified-antelope-centos9/openstack-neutron-server:current-podified
   replicas: 1
-  secret: neutron-secret
+  secret: neutron-secret
```

Comment:
No change in this file so can checkout from previous commit to avoid showing it here.